Automattic/wp-calypso
# Domains: Update all domains view when there are no domains

#### Changes proposed in this Pull Request

This adds a custom prompt to buy domains when there are no domains listed yet in the all domains view.
<img width="929" alt="Screenshot 2020-07-22 at 7 11 29 PM" src="https://user-images.githubusercontent.com/13062352/88266331-5b488f80-cccf-11ea-8b76-57bd64cfad39.png">
#### Testing instructions

* For an account with no domains, go to /domains/manage
* Verify you see the view shown in the screenshot above
* Ensure things work as expected when the user has any domains
